server/handlers/error.go

```diff
@@ -523,10 +525,18 @@ func ErrUpdateEvent(err error, id string) error {
 	return errors.New(ErrUpdateEventCode, errors.Alert, []string{fmt.Sprintf("Could not update event status for %s", id)}, []string{err.Error()}, []string{"Provided event status not supported", "Event has been deleted or does not exist", "Database is corrupt."}, []string{"Verify event filter settings", "Reset database."})
 }
 
+func ErrBulkUpdateEvent(err error) error {
+	return errors.New(ErrBulkUpdateEventCode, errors.Alert, []string{"Could not update status for events associated with provided ids"}, []string{err.Error()}, []string{"Provided event status not supported", "Event has been deleted or does not exist", "Database is corrupt."}, []string{"Verify event filter settings", "Reset database."})
+}
+
 func ErrDeleteEvent(err error, id string) error {
 	return errors.New(ErrDeleteEventCode, errors.Alert, []string{fmt.Sprintf("Could not delete event %s", id)}, []string{err.Error()}, []string{"Event might have been deleted and doesn't exist", "Database is corrupt."}, []string{"Verify event filter settings", "Reset database."})
 }
 
+func ErrBulkDeleteEvent(err error) error {
+	return errors.New(ErrBulkDeleteEventCode, errors.Alert, []string{"Could not delete events associated with provided ids"}, []string{err.Error()}, []string{"Provided events might have been deleted and doesn't exist", "Database is corrupt."}, []string{"Verify event filter settings", "Reset database."})
+}
+
 func ErrUnsupportedEventStatus(err error, status string) error {
 	return errors.New(ErrUnsupportedEventStatusCode, errors.Alert, []string{fmt.Sprintf("Event status %s not supported.", status)}, []string{err.Error()}, []string{"Unsupported event status for the current Meshery Server."}, []string{"Try upgrading Meshery to latest version.", "Use one of the supported event statuses."})
```

Comment:
```suggestion
	return errors.New(ErrUnsupportedEventStatusCode, errors.Alert, []string{fmt.Sprintf("Event status '%s' is not a supported status.", status)}, []string{err.Error()}, []string{"Unsupported event status for your current version of Meshery Server."}, []string{"Confirm that the status you are using is valid and a supported event status. Refer to Meshery Docs for a list of event statuses.", "Check for availability of a new version of Meshery Server. Try upgrading to the latest version." })
```

@MUzairS15, please be sure to put the probable cause and the suggested remediation in proper sequence. Understand that the array indexes must align between the two.